frameworks/Swift/vapor/Sources/TfbCommon/ContentMiddleware.swift

```diff
@@ -1,19 +1,19 @@
-import HTTP
+import Vapor
 
 /// Middleware that adds `Server` HTTP header to response.
 public final class ContentMiddleware: Middleware {
```

Comment:
This middleware shouldn't be necessary... I think it might have been added due to a bug in Vapor 2.